zebedee-cms/src/test/java/com/github/onsdigital/zebedee/ZebedeeTestBaseFixture.java

```diff
@@ -78,10 +84,20 @@ public void init() throws Exception {
         when(usersService.list())
                 .thenReturn(usersList);
 
+        Session session = new Session();
+        session.setEmail(builder.publisher1.getEmail());
+        session.setId("1234");
+        session.setLastAccess(new Date());
+        session.setStart(new Date());
+
+        when(newSessionsService.create(any(User.class))).thenReturn(session);
+        when(newSessionsService.get(anyString())).thenReturn(session);
+        when(newSessionsService.find(anyString())).thenReturn(session);
```

Comment:
Nit picky but this should probably specify the string value to return a session for:

````
when(newSessionsService.find(eq("1234")).thenReturn(session);
````
It ensures the test is providing the expected session to this mock call.

Reply:
There is one test that breaks when I do this and I can't figure out how to fix it. Not sure if you want to take a look?

Reply:
That test case that is failing is passing in a different ID value so the mock case is not matching. I didn't notice this was the test base class so it probably shouldn't be too specific so you can leave it as it is.